src/ontology/OntoFox_inputs/UO_input.txt

```diff
@@ -16,6 +16,9 @@ http://purl.obolibrary.org/obo/UO_0000105
 http://purl.obolibrary.org/obo/UO_0000109
 http://purl.obolibrary.org/obo/UO_0000270
 http://purl.obolibrary.org/obo/UO_0000280
+http://purl.obolibrary.org/obo/UO_0000061 # unit of molarity for TURBO assays

```

Comment:
These three should be in just one of these UO files. It just depends on whether we want it in OBI as a class or as an instance.